scripts/js/footer.js

```diff
@@ -556,7 +562,8 @@ function updateVersionInfo() {
           dockerUpdate = true;
         }
 
-        if (updateComponentAvailable) {
+        // Display update information of individual components only if we are not running in a Docker container
+        if ((!isDocker || (isDocker && v.name === "Docker Tag")) && updateComponentAvailable) {
```

Comment:
What's the magic behind this special hard-coded string `"Docker Tag"`? Is this what locally built containers show? If so (or, actually, in any case), we should leave some reasoning in a comment here. Especially as this string wasn't there in v5 before, either: https://github.com/pi-hole/web/blob/5c8945c81c98f6f17e5c138ffbfdcb0831e5a062/scripts/pi-hole/php/update_checker.php#L82-L100

Reply:
It's the name we set just a few lines up

https://github.com/pi-hole/web/blob/c689fdc72c3bdf7107d5d7d2b1c9003321c2d368/scripts/js/footer.js#L486

Reply:
Ah, I see. I somehow did not even try to search the `web` repo but everything else :-)

I thought we use `updateComponentAvailable` only for core, web, and FTL but we also use it for "the docker component". I was maybe confused because we also have `dockerUpdate` but this only serves the purpose of showing the footer text to refresh the container.